ui/panels-plugin/src/plugins/time-series-chart/VisualOptionsEditor.test.tsx

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { render, screen, fireEvent } from '@testing-library/react';
+import { VisualOptions } from './time-series-chart-model';
+import { VisualOptionsEditor } from './VisualOptionsEditor';
+
+describe('VisualOptionsEditor', () => {
+  const renderVisualOptionsEditor = (value: VisualOptions, onChange = jest.fn()) => {
+    render(<VisualOptionsEditor value={value} onChange={onChange} />);
+  };
+
+  const getLineWidthSlider = () => {
+    return screen.getByTestId('slider-line-width');
```

Comment:
should we have constants for all test id so if we ever need to update it, we don't have to update in all places?